src/actions/google/drive/sheets/google_sheets.ts

```diff
@@ -14,7 +14,9 @@ const MAX_REQUEST_BATCH = process.env.GOOGLE_SHEETS_WRITE_BATCH ? Number(process
 const MAX_ROW_BUFFER_INCREASE = 6000
 const SHEETS_MAX_CELL_LIMIT = 5000000
 const MAX_RETRY_COUNT = 5
+const RETRY_BASE_DELAY = process.env.GOOGLE_SHEETS_BASE_DELAY ? Number(process.env.GOOGLE_SHEETS_WRITE_BATCH) : 3
```

Comment:
Should this number be GOOGLE_SHEETS_BASE_DELAY as well

Reply:
No. The env var is more specific in case I implement it for other actions.

Reply:
Should this be its own envar then?

Reply:
I meant specifically on this line we reference two env vars, I figured they should be the same env var